components/Listen/EpisodeList/EpisodeCard/EpisodeCard.tsx

```diff
@@ -122,27 +121,24 @@ const EpisodeCard = ({ index, episode, onEpisodeClick }: IEpisodeCardProps) => {
                 <IconButton
                   className={styles.pause}
                   onClick={handlePauseButtonClick}
-                  aria-label="Pause"
+                  title="Pause This Episode"
                 >
                   <PauseCircleIcon />
                 </IconButton>
               ) : (
                 <IconButton
                   className={styles.play}
                   onClick={handlePlayButtonClick}
-                  aria-label="Play"
+                  title="Play This Episode"
                 >
                   <PlayCircleIcon />
                 </IconButton>
               )}
             </span>
 
             {explicit && (
-              <span className={styles.explicit}>
-                <ExplicitIcon
-                  className={styles.explicit}
-                  aria-label="Explicit"
-                />
+              <span className={styles.explicit} aria-label="Explicit">
```

Comment:
Lets move this `aria-label` back to the `<ExplicitIcon>` element.

Reply:
Moved it back to the `ExplicitIcon` element on line 141; is there a separate `ExplicitIcon` component?